compiler/parser/RdrHsSyn.hs

```diff
@@ -2284,12 +2289,18 @@ validateInterface name requires decls = do
     (e:es) -> do
       mapM_ (\e' -> report (getLoc e') "Multiple 'ensure' declarations") es
       pure (Just e)
+  viViewType <- case ibdViewDecls of
+    [] -> pure Nothing
```

Comment:
Do we nod need to report an error here similar to how we report an error for a missign signatory declaration?

Reply:
fair point, maybe add a TODO to fix that once you’ve adjusted all tests?

Reply:
Done